pkg/network/network.go

```diff
@@ -155,6 +185,26 @@ func NewConfigFromConfigMap(configMap *corev1.ConfigMap) (*Config, error) {
 		nc.DomainTemplate = dt
 	}
 
+	if autoTLS, ok := configMap.Data[AutoTLSKey]; !ok {
+		nc.AutoTLS = false
+	} else {
+		nc.AutoTLS = strings.ToLower(autoTLS) == "enabled"
+	}
+
+	if httpProtocol, ok := configMap.Data[HTTPProtocolKey]; !ok {
```

Comment:
same here, just do a switch over the `Data[key]` and add a case for `""`.

Reply:
Done